### DISCUSSION TITLE
After restoring the game, the items in the trash cans appear again. This also applies to abandoned outpost's closets.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#15230] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After restoring the game, the items in the trash cans reappear.

### Reproduction steps

Collect the items from the trash.
Save the game
Restore the game
Collect the same items from the same trash cans.
And so on ad infinitum.
This also applies to abandoned outposts.
You can undock, dock and loot the outpost again. Especially if the Security Officer has the Scavenger perk.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SAYtan47** [comment]
Just discovered this bug myself. Still exists on 1.9.7 version of the game

**Comment by SoobGorl** [comment]
This no longer occurs, tested in current release + internal branch. Your dumpster diving days are over.
